package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mysterium-tequilapi",
-  "version": "0.8.1",
+  "version": "0.8.2",
```

Comment:
This version was already released by @ignasbernotas https://github.com/mysteriumnetwork/js-tequilapi/pull/57/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2
Bump!